deploy/docker/common/nginx-config.conf

```diff
@@ -11,6 +11,16 @@ server {
     gzip_buffers 16 8k;
     gzip_http_version 1.1;
 
+    # to handle uri issue 414 from nginx
+    fastcgi_buffers 8 16k;
+    fastcgi_buffer_size 32k;
+
+    client_max_body_size 24M;
```

Comment:
Resolves the 413 (Request Entity Too Large) error if in case we would have exceeded default `1M` body size.

Reply:
Yeah, I suppose we can keep it since it is just an upper limit on the max client body size.